### PR TITLE
Discord link: Rename to discord and use working link

### PR DESCRIPTION
**Pull request by JJamesWWang**

### CODE REVIEW

**Review by JiaqiWang18 — APPROVED** [review]

File: static/js/redux/ui/semesterly.tsx [hunk 1]
```diff
@@ -223,11 +223,9 @@ const Semesterly = () => {
                 <a
                   target="_blank"
                   rel="noopener noreferrer"
-                  href="https://discord.gg/txYbphsAV7"
-                  // TODO: add discord logo correctly
+                  href="https://discord.gg/HmnwdbehBh"
                 >
-                  <i className="fab fa-discord" />
```
Comment (yesterd3v) [inline static/js/redux/ui/semesterly.tsx:229]: Did you intend to remove the Discord logo?

**Review by yesterd3v — CHANGES_REQUESTED** [review]

File: static/js/redux/ui/semesterly.tsx (hunk 1 above)
Comment (yesterd3v) [reply static/js/redux/ui/semesterly.tsx:229]: Ah the tag never worked, I forgot. Leave the TODO though

Comment (JJamesWWang) [reply static/js/redux/ui/semesterly.tsx:229]: I'll make it an issue

Comment (yesterd3v) [reply static/js/redux/ui/semesterly.tsx:229]: Okay

**Review by yesterd3v — APPROVED** [review]

**Pull request merged**